PySDM/builder.py

```diff
@@ -45,6 +45,12 @@ def add_dynamic(self, dynamic):
         assert key not in self.particulator.dynamics
         self.particulator.dynamics[key] = dynamic
 
+    def remove_dynamic(self, dynamic):
+        assert self.particulator.environment is not None
+        key = inspect.getmro(type(dynamic))[-2].__name__
```

Comment:
@edejong-caltech let's avoid code duplication with `add_dynamic` and extract the key-generation line into a separate method?